client/src/features/bulletins/Bulletins/Filters/DoubleSlider/DoubleSlider.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useRef, useEffect } from 'react';
+import './DoubleSlider.scss';
+import { CustomInput } from 'reactstrap';
+
+type Props = {
+  setMinPrice: (minPrice: number) => void;
+  setMaxPrice: (maxPrice: number) => void;
```

Comment:
I nomi degli eventi hanno un prefisso `on-`, tipo `onMinChange`. Il fatto che tu l'abbia chiamato `setMinPrice` denota ancora di più che proprio non hai capito il modo di pensare in React a Componenti